addons/hr_timesheet/models/project.py

```diff
@@ -301,6 +302,24 @@ def _compute_progress_hours(self):
                 task.progress = 0.0
                 task.overtime = 0
 
+    @api.depends('planned_hours', 'remaining_hours')
+    def _compute_remaining_hours_percentage(self):
+        for task in self:
+            if task.planned_hours > 0:
+                task.remaining_hours_percentage = task.remaining_hours / task.planned_hours
```

Comment:
thank you for your review 
why comparing with 0.0 instead of 0 
and why not putting False instead of task.planned_hours > 0.0 as we're on the else and we know it's false all time

Reply:
Indeed you can directly put False. 🙂 

- *why comparing with 0.0 instead of 0* 
It is not mandatory but it is to compare a Float with a Float. 🙂